### ISSUE TITLE
Tag list and definition list rules as WCAG 4.1.1

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#1020]
Please consider tagging the following rules as WCAG 4.1.1 in addition to 1.3.1 as these rules look at nesting errors in the content implemented using markup language and [SC of WCAG 4.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html) states 
> In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features. 

#### Rules list:
1. definition-list
2. dlitem
3. list
4. Listitem

**Comment by WilcoFiers** [comment]
My understanding of this "elements are nested according to their specifications" is that it means that elements that require a closing tag have one, that elements that don't allow a closing tag don't have them, and that tags are closed in the correct order.

That's inferred from https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140408/H74

**Comment by iamrafan** [comment]
Sure, this might be more related to https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H88
, specifically to "**Using features in the manner prescribed by the specification**"

**Comment by WilcoFiers** [comment]
I suppose so. Thing is, sufficient techniques are not a requirement. Following the HTML spec is one way to ensure content meets the parser requirements. But the technique goes well beyond what's minimally required for WCAG, including a bunch of stuff that's (IMO) part of SC 1.3.1.

**Comment by iamrafan** [comment]
If we map them to both 1.3.1 & 4.1.1, are you concerned that:
- A pass case means the successful implementation of SC for WCAG 1.3.1 and WCAG 4.1.1
- But a fail case would not necessarily mean a WCAG 4.1.1 failure?

**Comment by iamrafan** [comment]
@WilcoFiers sure, we agree.

**Comment by robfentress** [comment]
So, I just want to make sure I'm understanding things correctly: You are saying that the phrase, "elements are nested according to their specifications" does _not_ mean that an element needs to have any required children or parents as defined in the the markup language's specification, but, _rather_, that it is addressing the case where you might have the following:
```
<ul>
<li>list item</li>
<li> another list item</ul>
</li>
```
Is that what you're positing?
